### PR TITLE
Enable "Export invisible Elements to MusicXML"

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
as an advanced preference (it isn't available in the regular MusicXML export dialog)

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
I don't understand why the vtest current build fails, but is does seem unrelated to my changes.
```
fatal error: file '/home/runner/work/MuseScore/MuseScore/build.debug/src/framework/global/CMakeFiles/global.dir/cmake_pch.hxx' has been modified since the precompiled header '/home/runner/work/MuseScore/MuseScore/build.debug/src/framework/global/CMakeFiles/global.dir/cmake_pch.hxx.pch' was built: mtime changed
note: please rebuild precompiled header '/home/runner/work/MuseScore/MuseScore/build.debug/src/framework/global/CMakeFiles/global.dir/cmake_pch.hxx.pch'
1 error generated.
```

File: src/importexport/musicxml/internal/musicxmlconfiguration.cpp [hunk 1]
```diff
@@ -45,6 +46,9 @@ void MusicXmlConfiguration::init()
     settings()->setDefaultValue(MUSICXML_EXPORT_LAYOUT_KEY, Val(true));
     settings()->setDefaultValue(MUSICXML_EXPORT_BREAKS_TYPE_KEY, Val(MusicxmlExportBreaksType::All));
     settings()->setDefaultValue(MUSICXML_EXPORT_INVISIBLE_ELEMENTS_KEY, Val(false));
+    settings()->setDescription(MUSICXML_EXPORT_INVISIBLE_ELEMENTS_KEY, muse::qtrc("project/export",
```
Comment (RomanPudashkin) [inline src/importexport/musicxml/internal/musicxmlconfiguration.cpp:49]: It should be "iex_musicxml", not "project/export". You should also use muse::trc here

Comment (Jojo-Schmitz) [reply src/importexport/musicxml/internal/musicxmlconfiguration.cpp]: done

Comment (Jojo-Schmitz) [reply src/importexport/musicxml/internal/musicxmlconfiguration.cpp]: doesn't work. See also a few lines up, using the same qtrc and the same context

Comment (Jojo-Schmitz) [reply src/importexport/musicxml/internal/musicxmlconfiguration.cpp]: hold on, need to remove the `toStdString()` for the trc to work. 
Will change the other one too, as well the the context for both.

**Review by RomanPudashkin — APPROVED** [review]

**Comment by Jojo-Schmitz** [comment]
Wanted for 4.3.0?

**Comment by RomanPudashkin** [comment]
Please don't port your PRs to 4.3 unless you ask you

**Pull request merged**